pom.xml

```diff
@@ -317,7 +318,6 @@
                     <artifactId>maven-dependency-plugin</artifactId>
                     <version>${version.dependency.plugin}</version>
                     <configuration>
-                        <!-- Perfs improvements - PAR-191 -->
```

Comment:
this also shouldn't be removed

Reply:
okay I will add them back and again do the pull request

Reply:
no, this comment is tight to the next line, so it should be in here.

Reply:
Okay